simapp/simd/cmd/cmd_test.go

```diff
@@ -0,0 +1,22 @@
+package cmd_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/cosmos-sdk/simapp/simd/cmd"
+	"github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
+)
+
+func TestInitCmd(t *testing.T) {
+	cmd.RootCmd.SetArgs([]string{
```

Comment:
Since there's only one singleton RootCmd, this might not work if we add more integration tests on RootCmd in this file (i think).

Lmk if you can think of other ways to test RootCmd (one idea: remove singleton, use a function NewRootCmd).

Reply:
Please let's remove the singleton!